README.md

```diff
@@ -14,6 +14,8 @@ Make sure you have the regular Airbnb config setup. See [eslint-config-airbnb](h
 
 ```bash
 npm install eslint-config-airbnb-typescript \
+            eslint@^8.0.0 \
+            eslint-plugin-import@^2.0.0 \
```

Comment:
Not sure we need to add these here, as these deps would already be installed in Step 1 (above)